app/models/miq_server/worker_management/monitor.rb

```diff
@@ -166,8 +166,7 @@ def sync_monitor
   end
 
   def key_store
-    require 'dalli'
-    @key_store ||= Dalli::Client.new(MiqMemcached.server_address, :namespace => "server_monitor")
+    @key_store ||= MiqMemcached.client("server_monitor")
```

Comment:
I love this refactoring of the Memcached client...wonder if that should just be in a PR all on its own, since it's unrelated to Redis.